database-restore-postgres.sh

```
@@ -0,0 +1,289 @@
+}
+
+# Function to handle errors
+handle_error() {
+    log "ERROR: $1"
+    exit 1
+}
+
+# Load environment variables from .backup file if present
+dir="$(dirname "$(realpath "$0")")"
+if [[ -f "$dir/.backup" ]]; then
+    export $(grep -v '^#' "$dir/.backup" | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)